Functions/xASL_bids_Dicom2JSON.m

```diff
@@ -105,10 +105,23 @@
 	% ----------------------------------------------------------------------------------
 	for iJSON = 1:length(pathJSON)
 		if ~isempty(pathJSON{iJSON})
-			if imPar.bVerbose; fprintf('Recreating parameter file: %s\n',pathJSON{iJSON}); end
+            
+            % Print file name
+            if isfield(imPar,'TempRoot') && ~isempty(strfind(pathJSON{iJSON},imPar.TempRoot))
+                % Relative path
+                printFileName = pathJSON{iJSON}(length(imPar.TempRoot)+1:end);
```

Comment:
Michael?

Reply:
Sorry guys, was just working on the print design. Doesn't really affect the pipeline at all. It just nicely prints the file paths so that everything stays readable. Looks really nice IMHO if you run DCM2NII now 😄

Reply:
OK nice!